src/sandbox.rkt

```diff
@@ -38,10 +38,34 @@
       ([(pt ex) (in-pcontext context)])
     (values pt ex)))
 
+(define (get-sample test #:seed [seed #f] #:profile [profile? #f])
```

Comment:
The problem is that we're not checking or handling timeouts or exceptions.We could fix both by leveraging `get-test-result` to do the timeout/exception handling. There's also a potential refactoring to _call_ this from `run-herbie` instead of having duplicate code. If you want I can do these changes.

Reply:
If you want to handle these changes, that would be great!

Reply:
I can do that, ping me when the other stuff is done.

Reply:
@pavpanchekha Sorry this took a while, the other stuff is done!